processor/configuration/src/main/java/com/redhat/service/smartevents/processor/GatewayResolver.java

```diff
@@ -0,0 +1,9 @@
+package com.redhat.service.smartevents.processor;
+
+import com.redhat.service.smartevents.infra.models.gateways.Action;
+import com.redhat.service.smartevents.infra.models.gateways.Gateway;
+
+public interface GatewayResolver<T extends Gateway> extends GatewayBean<T> {
+
+    Action resolve(T gateway, String customerId, String bridgeId, String processorId);
```

Comment:
I believe this interface is not properly designed.

`SendToBridgeActionResolver` is not using `processorId` and `SlackActionResolver` is not using `customerId` and `bridgeId`. Methods with more than 3 parameters decrease the code readability. Could we please refactor this interface?

Reply:
Created https://issues.redhat.com/browse/MGDOBR-632